vectoriadb-index/src/test/java/jetbrains/vectoriadb/index/L2PQKMeansTest.java

```diff
@@ -132,6 +131,105 @@ public void testPQKMeansQuality() throws Exception {
         }
     }
 
+    @Test
+    public void randomClusterInitializationVsKmeansPpBenchmark() throws Exception {
```

Comment:
For this benchmark, I get results like
`meanSilhCoeffDiff: 0,006831, meanSilhCoeffRandom: 0,098650, meanSilhCoeffPP: 0,105481`

Reply:
So it seems that k-means++ does a bit better.